services/secret-service/src/route/auth-clients/index.js

```diff
@@ -26,6 +27,13 @@ const authClientObfuscator = {
         clientId: '***',
         clientSecret: '***',
     }),
+
+    [OA2_PASSWORD]: (value) => ({
+        ...value,
+        clientId: '***',
+        clientSecret: '***',
+        password: '***',
```

Comment:
Does fullResponse need added to this?

Reply:
`fullResponse` is only on the secret, not on the auth client, and I believe these values only apply to auth clients.